core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1279,9 +1279,29 @@ secret_configs:
         namespace: ci-release-s390x-priv
         name: git-credentials
   - from:
+      # release-controllers - kubeconfigs (app.ci)
       sa.release-controller.app.ci.config:
         bw_item: release-controller
         attachment: sa.release-controller.app.ci.config
+      sa.release-controller-ocp.app.ci.config:
+        bw_item: release-controller
+        attachment: sa.release-controller-ocp.app.ci.config
```

Comment:
These secrets don't exist, the only `app.ci` one is `sa.release-controller.app.ci.config`

Reply:
Hey @alvaroaleman
Thanks, I was afraid of that.  Any idea how these entries got created in the first place?  I was hoping by the `ci-secret-generator`, but it doesnt have an entry for `release-controller` or anything that might have created them.
The service accounts exist on api.ci, would it make sense to add them to the generator?
Thanks for your help!

Reply:
Okay I've created https://github.com/openshift/release/pull/12869 which includes both your changes here and config for the `secret-generator` to generate these. The latter requires the secret to actually be used, that's why I included your commit in my PR, hope thats okay

Reply:
That's perfectly fine!  I just want to make sure it's done right and accounted for if/when we need to migrate again.